dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -414,6 +414,25 @@ public boolean terminateUser(
         return true;
     }
 
+    @PATCH
+    @Timed
+    @UnitOfWork
+    @Path("/user/{userId}")
+    @RolesAllowed("admin")
+    @Operation(operationId = "reactivateUser", description = "Reactivate a user that has been terminated", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "Reactivate a user that has been terminated", hidden = true)
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "Successfully reactivated user", content = @Content(schema = @Schema(implementation = Boolean.class)))
+    @ApiResponse(responseCode = HttpStatus.SC_FORBIDDEN + "", description = HttpStatusMessageConstants.FORBIDDEN)
+    @ApiResponse(responseCode = HttpStatus.SC_NOT_FOUND + "", description = USER_NOT_FOUND_DESCRIPTION)
+    public boolean reactivateUser(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user") @Auth User authUser,
```

Comment:
From a RESTful purist-ish perspective:

1. You should return the user, not a boolean. The HTTP status code will indicate success or failure.
2. I don't think you should take up `/user/{userId}` for this feature alone

You could instead create a `UserLite` (I wouldn't use that name) class that so far only has one optional property, `banned`. Then if we choose to allow additional editable things, we just add to that class.

On the other hand, this is an endpoint that we'll hopefully never have to invoke, at least in its current incarnation, so, worth the effort? I don't know.